package BASIC_JAVA_CONTENT.question.pattern;

import java.util.Scanner;

public class pattern2 {
    public static void main(){
        Scanner sc = new Scanner(System.in);
        int side=sc.nextInt();
        for (int i = 1; i <= side; i++) {
            for (int j = 1; j <= side; j++) {
                System.out.print(j+" ");
            }
            System.out.println();
        }
    }
}
